pixivpy3/api.py

```diff
@@ -78,20 +87,23 @@ def set_client(self, client_id, client_secret):
         self.client_id = client_id
         self.client_secret = client_secret
 
-    def auth(self, username=None, password=None, refresh_token=None, headers={}):
+    def auth(self, username=None, password=None, refresh_token=None, headers=None):
         """Login with password, or use the refresh_token to acquire a new bearer token"""
         local_time = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S+00:00')
+        headers = CaseInsensitiveDict(headers or {})
         headers['x-client-time'] = local_time
         headers['x-client-hash'] = hashlib.md5((local_time + self.hash_secret).encode('utf-8')).hexdigest()
         # Allow mock UA due to #171: https://github.com/upbit/pixivpy/issues/171
-        if headers.get('User-Agent', None) == None and headers.get('user-agent', None) == None:
+        if 'user-agent' not in headers:
```

Comment:
Is the key of the map case sensitive?
`if 'user-agent' not in headers and if 'User-Agent' not in headers:`

Reply:
`CaseInsensitiveDict`, got it